02-Project-Requirements/.travis.yml

```diff
@@ -1 +1,10 @@
 language: node_js
+node_js: "8"
+branches:
+  only:
+    - master
+cache:
+  directories:
+    - node_modules
+before_install:
+  - mysql -e 'CREATE DATABASE testdb;'
```

Comment:
this will need to get deleted. For some reason this line was there by default but needs to be deleted.
```suggestion

```